src/client/ds3/pool.c

```diff
@@ -0,0 +1,175 @@
+/**
+ * (C) Copyright 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#include "ds3_internal.h"
+#include <sys/stat.h>
+#include <fcntl.h>
+
+int
+ds3_init(void)
+{
+	return -dfs_init();
+}
+
+int
+ds3_fini(void)
+{
+	return -dfs_fini();
+}
+
+/**
+ * Helper functions for metadata directories management
+ */
+
+const char *
+meta_dir_name(enum meta_dir dir)
+{
+#define X(a, b)                                                                                    \
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  Macros with flow control statements should be avoided
(style)  macros should not use a trailing semicolon